AppCenter/AppCenter/Internals/Model/MSAbstractLog.m

```diff
@@ -101,17 +101,17 @@ - (NSString *)serializeLogWithPrettyPrinting:(BOOL)prettyPrint {
   return jsonString;
 }
 
-#pragma mark - Tenant
+#pragma mark - Transmission Target logic
 
-- (NSSet *)getTenants {
+- (NSSet *)getTransmissionTargets {
```

Comment:
I think we get the `getTransmissionTargetTokens` here not the target objects, we should add tokens to the name?

Reply:
I wonder if this one is already specified or not.